packages/eslint-config-nori/eslint-local-rules/waffle-as-promised.js

```diff
@@ -0,0 +1,37 @@
+const findInExpressionTree = require('@fintechstudios/eslint-plugin-chai-as-promised/lib/util/find-in-expression-tree'); // todo try replacing with esquery
+
+const isWaffleAsPromised = (node) => {
+  return node.type === 'MemberExpression' &&
+    ['emit'].includes(node.property.name)
+    ? node.property
+    : undefined;
+};
+
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description:
+        'Must handle promises returned from waffle-as-promised expressions',
```

Comment:
Is it possible to add the pass/fail example in the description to this bit?

Reply:
Or actually whichever bit shows in intellisense

Reply:
not sure, tbh, this is my first custom eslint rule. Do you spot where that might go [here](https://eslint.org/docs/latest/developer-guide/working-with-rules)? 

What you're asking might be more about what's in `context.report` though. Happy to add this if you can expand a bit more on what you're asking!